source/getting_started/step-by-step.rst

```diff
@@ -1,10 +1,55 @@
 .. _step-by-step:
 
+
+.. |checkbox| raw:: html
+
+    <input type="checkbox">
+
+
 ============
 Step by step
 ============
 
 
+Checklist
+=========
+
+Setting up ETOS is a daunting task when you are doing it the first time around.
+But it is also easy to miss a few steps that are required for it to work.
+
+For this reason we have created a checklist in order to keep track of what needs to be done so that no steps are forgotten.
+
+* |checkbox| Set up a Kubernetes cluster (ETOS runs here)
+* |checkbox| Set up an execution space (Execute test runner containers)
```

Comment:
It might not be obvious what an execution space is. If there is some section in the ETOS docs that explain it should be linked here.

Reply:
Done